brightlayer-ui/angular-workflows
# Bug/178 duplicate ids

Fixes #178 


#### Changes proposed in this Pull Request:
- Remove duplicate ID from workflow.
- Update cypress tests to pass.


#### To Test:
- `cd login-wrokflow && yarn cytest`, ensure the cypress tests are passing
- Also verify no console warnings are occurring on the Change Password dialog, Create Account password page, nor Reset Password page using the firebase deployed site.
